.vib/drupal/goss/vars.yaml

```diff
@@ -0,0 +1,24 @@
+binaries:
+  - apachectl
+  - gosu
+  - mysql
+  - php
+  - render-template
+  - drush
+files:
+  - paths:
+    - /opt/bitnami/drupal/sites/default/settings.php
```

Comment:
We should also check the custom Drupal config performed at the postunpack stage is properly set:

https://github.com/bitnami/containers/blob/91b3f028c95686ef8112f748d39c63fb20871164/bitnami/drupal/10/debian-11/rootfs/opt/bitnami/scripts/drupal/postunpack.sh#L50

Something similar happens with the changes to the following files:

- [PHP config file](https://github.com/bitnami/containers/blob/91b3f028c95686ef8112f748d39c63fb20871164/bitnami/drupal/10/debian-11/rootfs/opt/bitnami/scripts/drupal/postunpack.sh#L54) `/opt/bitnami/php/etc/php.ini`
- [Drupal .htaccess file](https://github.com/bitnami/containers/blob/91b3f028c95686ef8112f748d39c63fb20871164/bitnami/drupal/10/debian-11/rootfs/opt/bitnami/scripts/drupal/postunpack.sh#L127) `/opt/bitnami/apache/conf/vhosts/htaccess/drupal-htaccess.conf`

We need to do these kind of checks to make sure our added logic works as expected and no changes will break them (files not existing anymore, components not correctly added, etc)

Reply:
Added extra tests verifying the content of `php.ini`, `drupal-htaccess.conf`, `settings.php` and `httpd.conf`